geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/pooling/ConnectionManagerImplTest.java

```diff
@@ -92,9 +92,18 @@ public void startShouldEatRejectedExecutionException() {
   @Test
   public void borrowConnectionThrowsWhenUsingExistingConnectionsAndNoConnectionsExist() {
```

Comment:
We are still testing same thing since there is no free connections in pool, since you can not define pool with 0 connections. If you want I can change name ...AndNoFreeConnectionsExist

Reply:
Looking at the old code vs the new, it seems like originally we just tried to call `borrowConnection()` and asserted that we throw an exception, but in the new test we first successfully call the single-argument `borrowConnection()` and then call the three-arg version and assert that that throws. Given that there are two calls to `borrowConnection()` in the test and only one of them throws, a test name that explains this difference in behaviour between the calls would be best.